crypto/x509/x509name.c

```diff
@@ -334,14 +338,14 @@ int X509_NAME_ENTRY_set_data(X509_NAME_ENTRY *ne, int type,
     return (1);
 }
 
-ASN1_OBJECT *X509_NAME_ENTRY_get_object(X509_NAME_ENTRY *ne)
+const ASN1_OBJECT *X509_NAME_ENTRY_get0_object(const X509_NAME_ENTRY *ne)
```

Comment:
Sorry, I must have missed this before. We can't make this change. `X509_NAME_ENTRY_get_object` exists in 1.0.2, so we can't just delete it. We can add the `X509_NAME_ENTRY_get0_object` *as well* if we choose to.